Pipfile

```diff
@@ -1,23 +0,0 @@
-[[source]]
```

Comment:
Since we are removing the `Pipfile` we should remove the mention of it from the docs as well(`pipenv`): https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/contributing.md#L114

Reply:
I actually found a way to use `pipenv` with requirements (via `-r requirements.txt` during install). I think @priyolahiri is using this, so it might be helpful.